rmgpy/molecule/molecule.py

```diff
@@ -1118,6 +1118,19 @@ def remove_van_der_waals_bonds(self):
             if bond.is_van_der_waals():
                 self.remove_bond(bond)
 
+    def add_van_der_waals_bond(self):
+        """
+        Adds a single van der Waals bond to the graph to connect a surface site to the physisorbed absorbate.
+        """
+        cython.declare(bond=Bond,fragment=Molecule,fragments=list)
+
+        if self.contains_surface_site():
+            fragments = self.split()
+            if len(fragments) == 2:
+                if any([fragment.is_surface_site() for fragment in fragments]):
```

Comment:
If you remove the `[` and `]` from this line then the `any` will operate over the generator `fragment.is_surface_site() for fragment in fragments` and it'll be able to abort the loop as soon as one of them returns `True`. By creating a list first by wrapping the generator with `[ ]`, you force it to go to the end of the loop, and you add the overhead of creating the list object.

Here's an exaggerated example:
```python
In [3]: f = list(range(-3,100))

In [4]: any([i>0 for i in f])
Out[4]: True

In [5]: any(i>0 for i in f)
Out[5]: True

In [6]: %timeit any([i>0 for i in f])
3.9 µs ± 121 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [7]: %timeit any(i>0 for i in f)
503 ns ± 1.69 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

In this context it's not going to make a gigantic deal, but thought you might like to learn the trick :-)